graphene_django/views.py

```diff
@@ -112,7 +112,7 @@ def __init__(
         self.pretty = self.pretty or pretty
         self.graphiql = self.graphiql or graphiql
         self.batch = self.batch or batch
-        self.execution_context_class = execution_context_class
+        self.execution_context_class = execution_context_class or self.execution_context_class
```

Comment:
We probs want to follow the same pattern as the other attributes (`graphiql`, `batch` etc):

```suggestion
        self.execution_context_class = self.execution_context_class or execution_context_class
```

Reply:
I think so this is not a good pattern. class attribute it's something like default value. when we create an instance we need to have approach to redefine it. thats why i use current order.

Reply:
@Arfey Nice, thats a good point - could you update the other attributes to follow that pattern instead?

Reply:
done